src/test/java/teammates/test/pageobjects/InstructorCourseDetailsPage.java

```diff
@@ -123,57 +123,74 @@ public InstructorCourseDetailsPage clickRemindStudentAndConfirm(String studentNa
     
     public InstructorCourseDetailsPage clickDeleteAndCancel(String studentName) {
         int rowId = getStudentRowId(studentName);
-        clickAndCancel(getDeleteLink(rowId));
+        if (rowId > -1) {
+            clickAndCancel(getDeleteLink(rowId));
+        }
```

Comment:
Instead of silencing the error, maybe throw an exception here? If the rowId is -1, then the test was not written correctly